photon-client/src/views/SettingsViews/General.vue

```diff
@@ -49,22 +49,70 @@
           <th class="infoElem">
             Disk Usage
           </th>
+          <th class="infoElem">
+            <v-tooltip top>
+              <template v-slot:activator="{ on, attrs }">
+                <span
+                  v-bind="attrs"
+                  v-on="on"
+                > 
+                ⓘ CPU Active Throttling 
+                </span>
+              </template>
+              <span>
+                    LV: Low Supply Voltage <br>
+                    FRQ: ARM Core Frequency is Capped<br>
+                    THR: CPU Throttling Active<br>
+                    TEMP: CPU Over-Temperature<br>
+              </span>
+            </v-tooltip>
+          </th>
+          <th class="infoElem">
+            <v-tooltip top>
+              <template v-slot:activator="{ on, attrs }">
+                <span
+                  v-bind="attrs"
+                  v-on="on"
+                >
+                ⓘ CPU Prev Throttling 
+                </span>
+              </template>
+              <span>
+                    LV: Low Supply Voltage <br>
+                    FRQ: ARM Core Frequency is Capped<br>
+                    THR: CPU Throttling Active<br>
+                    TEMP: CPU Over-Temperature<br>
```

Comment:
Are these messages generic enough to apply to non-pi linux platforms?

Reply:
They are not - the reason codes are currently PI-specific